python/nav/jwtconf.py

```diff
@@ -11,32 +11,58 @@ class JWTConf(NAVConfigParser):
     """jwt.conf config parser"""
 
     DEFAULT_CONFIG_FILES = ('jwt.conf',)
+    NAV_SECTION = "nav"
 
     def get_issuers_setting(self):
-        issuers_settings = dict()
-        for section in self.sections():
-            try:
+        try:
+            external_settings = self._get_settings_for_external_tokens()
+            local_settings = self._get_settings_for_nav_issued_tokens()
+            external_settings.update(local_settings)
+            return external_settings
+        except ConfigurationError as error:
+            _logger.error('Error reading jwtconfig: %s', error)
+            return dict()
```

Comment:
If local settings call fails here, then exception clause triggers and it returns an empty dict. meaning even if the external settings were OK they will not be loaded, and vice versa. Is it fine for none of the config to work if theres an error, or should the part that works make it through?